pkg/publish/docker.go

```diff
@@ -240,12 +255,22 @@ func publishManifest(manifest string, images []string) error {
 	}
 	manifestRef, err := name.ParseReference(manifest)
 	if err != nil {
-		return fmt.Errorf("failed to parse %v: %v", manifestRef, err)
+		return "", fmt.Errorf("failed to parse %v: %v", manifestRef, err)
 	}
 	if err := remote.MultiWrite(map[name.Reference]remote.Taggable{manifestRef: index}, remote.WithAuthFromKeychain(authn.DefaultKeychain)); err != nil {
-		return fmt.Errorf("failed to push %v: %v", manifestRef, err)
+		return "", fmt.Errorf("failed to push %v: %v", manifestRef, err)
 	}
-	return nil
+	newImg, err := remote.Image(manifestRef, remote.WithAuthFromKeychain(authn.DefaultKeychain))
```

Comment:
On this case I don't think we should need remote.Image, the SHA will be the same before and after we push it afaik?